### PR TITLE
:recycle: replace turbo by mocha test runner

### PR DESCRIPTION
**Pull request by corinnekrych**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,3 +4,22 @@
...
 This service supports Forms functionality for applications running on OpenShift v2 MBaaS type.
+# Build
+
+```
+npm install
+```
+
+# Tests
+
+All the tests are in the "test/" directory. The cloud app is using moka as the test runner.
```
Comment (jcesarmobile) [inline README.md]: mocha

File: Gruntfile.js [hunk 2]
```diff
@@ -100,19 +92,6 @@ module.exports = function(grunt) {
           './node_modules/.bin/istanbul report',
```
Comment (jcesarmobile) [inline Gruntfile.js]: you don't need to run report, it will be automatically run

File: Gruntfile.js [hunk 3]
```diff
@@ -100,19 +92,6 @@ module.exports = function(grunt) {
...
           'echo "See html coverage at: `pwd`/coverage/lcov-report/index.html"'
```
Comment (jcesarmobile) [inline Gruntfile.js]: the automatic istanbul report will be in `cov-unit` instead of `coverage`

File: .gitignore [hunk 4]
```diff
@@ -1,3 +1,5 @@
 node_modules
 npm-debug.log
 .idea/
+cov-unit/
+coverage/
```
Comment (jcesarmobile) [inline .gitignore]: no need to ignore coverage folder if you don't run istanbul report (see other comments)

**Review by jcesarmobile — COMMENTED** [review]
I've made a few comments.

Also, why do you delete npm-shrinkwrap.json file?

I get multiple errors like `{ [Error: Cannot find module './build/Debug/DTraceProviderBindings'] code: 'MODULE_NOT_FOUND' }`, but it seems to work fine despite that.

**Comment by corinnekrych** [comment]
@jcesarmobile I've got you point for duplicate coverage reports, see last commit to remove it.
the error is due to  dtrace-provider (transitive dependencies with ultiple version) - not sure how to fix that one.

File: package.json [hunk 5]
```diff
@@ -40,7 +40,9 @@
   "main": "application.js",
   "scripts": {
     "test": "grunt test",
+    "unit": "grunt unit",
```
Comment (jcesarmobile) [inline package.json]: there is no grunt unit task

File: Gruntfile.js [hunk 6]
```diff
@@ -150,9 +127,8 @@ module.exports = function(grunt) {
   grunt.registerTask('accept', ['env:local', 'shell:accept']);
 
   // Coverate tasks
-  grunt.registerTask('coverage', ['shell:coverage_unit', 'shell:coverage_accept']);
+  grunt.registerTask('coverage', ['shell:coverage_unit']);
   grunt.registerTask('coverage-unit', ['shell:coverage_unit']);
```
Comment (jcesarmobile) [inline Gruntfile.js]: this does the same as `coverage`, so it could be removed.

**Review by jcesarmobile — COMMENTED** [review]
Added two new comments.
Also, you forgot one of the previous comments, the moka -> mocha on README.md.

**Comment by corinnekrych** [comment]
@jcesarmobile mind giving another 👁  
I've rebased and squashed commits.

**Review by jcesarmobile — APPROVED** [review]
LGTM

**Pull request merged**